app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -0,0 +1,438 @@
+<!DOCTYPE html>
+  <!--[if lt IE 9]><html class="lte-ie8" lang="<%= html_lang %>"><![endif]-->
+  <!--[if gt IE 8]><!--><html lang="<%= html_lang %>"><!--<![endif]-->
+  <head>
+    <meta charset="utf-8" />
+    <title><%= title %></title>
+
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag "application", media: "all", 'data-turbolinks-track': "reload" %>
+    <%= stylesheet_link_tag "print", media: "print" %>
+
+    <link rel="shortcut icon" href="<%= asset_path 'favicon.ico' %>" type="image/x-icon" />
+    <link rel="mask-icon" href="<%= asset_path 'govuk-mask-icon.svg' %>" color="#0b0c0c">
+    <link rel="apple-touch-icon" sizes="180x180" href="<%= asset_path "govuk-apple-touch-icon-180x180.png" %>">
+    <link rel="apple-touch-icon" sizes="167x167" href="<%= asset_path "govuk-apple-touch-icon-167x167.png" %>">
+    <link rel="apple-touch-icon" sizes="152x152" href="<%= asset_path "govuk-apple-touch-icon-152x152.png" %>">
+    <link rel="apple-touch-icon" href="<%= asset_path "govuk-apple-touch-icon.png" %>">
+
+    <meta name="theme-color" content="#0b0c0c" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <% #  Ensure that older IE versions always render with the correct rendering engine %>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    <% # The default og:image is added below :head so that scrapers see any custom metatags first, and this is just a fallback %>
+    <meta property="og:image" content="<%= asset_path "govuk-opengraph-image.png" %>">
+
+    <%= yield :head %>
+  </head>
+  <body class="gem-c-layout-for-public govuk-template__body">
+    <script>
+      document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
+    </script>
+
+    <%= render "govuk_publishing_components/components/skip_link", {
+      href: "#content"
+    } %>
+
+    <%= render "govuk_publishing_components/components/cookie_banner" %>
+
+    <% unless omit_header %>
+      <%= render "govuk_publishing_components/components/layout_header", {
+        environment: "public",
+        search: show_search,
+        # layout-header will always have border-bottom, unless the layout is full width
+        remove_bottom_border: full_width,
+      } %>
+    <% end %>
+    <div class="<%= "govuk-width-container" unless full_width %>"  id="content">
+      <%= yield :before_content %>
+      <main class="govuk-main-wrapper">
+        <%= yield %>
+      </main>
+    </div>
+
+    <%= render "govuk_publishing_components/components/feedback" %>
+
+    <% unless local_assigns[:hide_footer_links] %>
+      <%= render "govuk_publishing_components/components/layout_footer", {
+        with_border: true,
+        navigation: [
+          {
+            title: "Prepare for Brexit",
+            columns: 2,
+            items: [
+              {
+                href: "/business-uk-leaving-eu",
+                text: "Prepare your business or organisation for Brexit",
```

Comment:
2 questions. These may well be annoying lack-of-context questions so soz in advance:

- Hard-coding this navigation seems a little risky. Is there no way for content designers to set this via publishing/whatever it's called?
- Failing point 1, is there a locale file for this content to go in?

Reply:
Hi @owenatgov not annoying at all, you've actually brought up a good point here.

I think the reason why it's hard coded in here is because, well.... it was probably hard coded in the template it was copied over from. 😅

I am actually not sure if the components gem draws any content from any of the _publishing/whatever it's called_ 's. My feeling is that it does not.

See, generally we pass content through to the gem components from the rendering app (content which may or may not come from a publishing app, or the rendering app's localisation files, or be hardcoded in the rendering app etc).

I'm not totally sure how we should handle this case, but I can't see a reason why we couldn't move this data to the gem's locale file? I can check with team and get back to you on that.

Reply:
Currently there's no way of content designers changing the footer links, so we probably don't need to worry about that. We should definitely have text like this in a locale file though.

Something else I've noticed is that this is a lot of data to put in the view. It would be better to have that data object come from a helper method, to unclutter the template.

Reply:
@andysellick  I've discussed this with @maxgds, @injms and @owenatgov and we've concluded that moving this content into a locale file sounds like a good idea. However, team thinks it needs scoping and discussion with the community, so we decided to leave it as is for now and create a card to address it in the future: https://trello.com/c/Jrb7JV0B

Reply:
Fair enough. Can we move all the data into a helper method though?

Reply:
I'm sorry @andysellick, for some reason I missed this comment before. 
Yes, I am happy to move the data into a helper. However I'm not sure which helper this should be and how the data should be stored?... is there any example that you know of that I could use as a model for storing data into a helper method?

Reply:
I think probably a new one? Maybe in this directory: https://github.com/alphagov/govuk_publishing_components/tree/master/lib/govuk_publishing_components/presenters

Here's an example of a helper that returns a chunk of text to be reused in the accessibility criteria - I was thinking a similar method that simply returned the data for the footer.

https://github.com/alphagov/govuk_publishing_components/blob/master/app/models/govuk_publishing_components/shared_accessibility_criteria.rb

Reply:
@andysellick I've moved the footer data into a helper, hopefully it's correct. I did it before I saw your latest comment so it's not exactly the same format as the accessibility criteria example you shared, but it works.

When you get a moment would you mind giving this branch another once-over? Please let me know if I've missed any comments (sometimes they'll disappear and I'm not sure how to find them again 😅) or if you have any more feedback on this.

Reply:
Helper looks good, I think. Good job. Will do.